rpcs3/Emu/Cell/Modules/cellGem.cpp

```diff
@@ -479,10 +479,6 @@ struct gem_config_data
 	gem_config_data(utils::serial& ar)
 	{
 		save(ar);
-
```

Comment:
Why remove this?

Reply:
It's not needed

Reply:
Why is it not needed?

Reply:
Because ar is in reading state in a constructor

Reply:
Why is the function called save then?

Reply:
It recycles the function, but it doesn't save here